tests/review-pr-workflow.test.sh

```
@@ -2580,8 +2617,22 @@ L_RUN_PR=4271
+L_RESEARCH_DIR="$(printf '%s\n' "$L_FIXTURE_OUT" | sed -n 3p)"
 L_RUN_DIR="$(printf '%s\n' "$L_FIXTURE_OUT" | sed -n 4p)"
 
+# The head this fixture run stands on, recorded exactly the way the Phase 1
+# scope fence records it (#479) — through the typed writer, before EITHER entry
+# path is probed. Position is deliberate: L5 compares the two environments as
+# they are captured below, so a carrier absent from the fixture is a carrier L5
+# cannot see, and L7 needs a record that predates the shells it probes from.
+L7_SEED=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+L7_FIXED=bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+if [ -z "$L_RESEARCH_DIR" ] || ! env -i PATH="$PATH" HOME="${HOME:-$TMP}" bash -c \
+     '. "$1"; review_fleet_write_reviewed_head "$2" "$3"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)